Update show storage info query (#320)
memgraph/documentation: pages/configuration/server-stats.md

The result will contain the following fields:

| Field                        | Description                                                                                                                                                                                   |
|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| vertex_count                 | The number of stored nodes (vertices).                                                                                                                                                        |
| edge_count                   | The number of stored relationships (edges).                                                                                                                                                   |
| average_degree               | The average number of relationships of a single node.                                                                                                                                         |

> | memory_res                   | The amount of RAM used reported by the OS (in B, KiB, MiB, GiB or TiB).                                                                                                                       |

Reviewer: ```suggestion
| memory_res                   | The non-swapped physical RAM memory a task has used, reported by the OS (in B, KiB, MiB, GiB or TiB).                                                                                                                       |
```

| disk_usage                   | The amount of disk space used by the data directory (in B, KiB, MiB, GiB or TiB).                                                                                                             |

> | memory_tracked               | The amount of RAM allocated in system and tracked by the Memgraph (in B, KiB, MiB, GiB or TiB).<br/>For more info, check out the [memory control](/fundamentals/storage-memory-usage).        |

Reviewer: ```suggestion
| memory_tracked               | The amount of RAM allocated in the system and tracked by Memgraph (in B, KiB, MiB, GiB or TiB).<br/>For more info, check out the [memory control](/fundamentals/storage-memory-usage).        |
```

> | vm_max_map_count             | Number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1 [memory control](/fundamentals/storage-memory-usage).                                    |

Reviewer: ```suggestion
| vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1 [memory control](/fundamentals/storage-memory-usage).                                    |
```

> | vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1. </br> For more info, check out [memory control(/fundamentals/storage-memory-usage#control-memory-usage).                                    |

Reviewer: ```suggestion
| vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1. </br> For more info, check out [virtual memory section of the docs](/fundamentals/storage-memory-usage#virtual-memory).                                    |
```

> | allocation_limit             | The current allocation limit set for this instance (in B, KiB, MiB, GiB or TiB).<br/>For more info, check out the [memory control](/fundamentals/storage-memory-usage).                       |

Reviewer: ```suggestion
| allocation_limit             | The current allocation limit set for this instance (in B, KiB, MiB, GiB or TiB).<br/>For more info, check out the [memory control](/fundamentals/storage-memory-usage#control-memory-usage).                       |
```

> | global_isolation_level       | The current `global` isolation level.<br/>For more info, check out the [isolation levels](/fundamentals/transactions#isolation-levels).                                                       |

Reviewer: ```suggestion
| global_isolation_level       | The current `global` isolation level.<br/>For more info, check out [isolation levels](/fundamentals/transactions#isolation-levels).                                                       |
```

| session_isolation_level      | The current `session` isolation level.                                                                                                                                                        |
| next_session_isolation_level | The current `next` isolation level.                                                                                                                                                           |

> | storage_mode                 | The current storage mode.<br/>For more info, check out the [storage modes](/fundamentals/storage-memory-usage#storage-modes).                                                                 |

Reviewer: ```suggestion
| storage_mode                 | The current storage mode.<br/>For more info, check out [storage modes](/fundamentals/storage-memory-usage#storage-modes).                                                                 |
```
